ci/citest/acceptance-test/tests/cmd_console_test.go

```diff
@@ -23,3 +20,15 @@ func TestCmdConsole_ShowOption(t *testing.T) {
 	RunCmdWithTimeoutAndReportFail(t, 10, 5, "openvdc", "destroy", instance_id)
 	WaitInstance(t, 5*time.Minute, instance_id, "TERMINATED", nil)
 }
+
+func TestLXCCmdConsole_ShowOption(t *testing.T) {
+	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc")
+	instance_id := strings.TrimSpace(stdout.String())
+	runConsoleCmd(instance_id, t)
+}
+
+// func TestQEMUCmdConsole_ShowOption(t *testing.T) {
+// 	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/qemu")
+// 	instance_id := strings.TrimSpace(stdout.String())
+// 	runConsoleCmd(instance_id, t)
+// }
```

Comment:
I take it qemu console is being developed in another branch that uncomments this code?